packages/pangea-sdk/PangeaCyber.Net/Config.cs

```diff
@@ -40,37 +42,44 @@ public Config(string token, string domain)
 
         #region Public Properties
 
-        ///
+        /// <summary>Pangea API token.</summary>
         public string Token { get; set; }
 
-        ///
+        /// <summary>Pangea API domain.</summary>
         public string Domain { get; set; }
 
-        /// @deprecated set it on Service builder instead
+        /// <exclude />
+        [Obsolete("Set config ID when building service clients instead.")]
         public string ConfigID { get; set; } = default!;
 
-        ///
+        /// <summary>Pangea environment.</summary>
+        /// <remarks>
+        /// This is intended to facilitate SDK development and should not be touched in everyday usage.
+        /// </remarks>
         public string Environment { get; set; }
 
-        ///
-        public bool Insecure { get; set; } = default!;
+        /// <summary>Whether or not to perform requests via plain HTTP, as opposed to secure HTTPS.</summary>
+        public bool Insecure { get; set; }
 
-        ///
-        public TimeSpan ConnectionTimeout { get; set; } = default!;
+        /// <summary>Time span to wait before a request times out.</summary>
+        public TimeSpan ConnectionTimeout { get; set; }
 
-        ///
+        /// <summary>User-Agent string to append to the default one.</summary>
         public string CustomUserAgent { get; set; } = default!;
 
-        /// Enable queued request retry support
+        /// <summary>Whether or not queued request retries are enabled.</summary>
         public bool QueuedRetryEnabled { get; set; } = true;
 
-        /// Timeout used to poll results after 202 (in secs)
+        /// <summary>Timeout used to poll results after a HTTP/202 (in seconds).</summary>
```

Comment:
do you think we can make this slightly more clear versus `ConnectionTimeout`? maybe explain that ConnectionTimeout is for the underlying http connection, whereas this is for the application?

Reply:
Took a crack at this, I'm not completely sold on what I came up with but I think it's better?

Reply:
HTTP 202 is better